tools/monitoring/main.go

```diff
@@ -27,17 +28,22 @@ import (
 	"github.com/knative/test-infra/shared/mysql"
 	"github.com/knative/test-infra/tools/monitoring/config"
 	"github.com/knative/test-infra/tools/monitoring/mail"
+	"github.com/knative/test-infra/tools/monitoring/subscriber"
 )
 
 var (
 	dbConfig   mysql.DBConfig
 	mailConfig *mail.Config
+	subClient  *subscriber.Client
 
 	alertEmailRecipients = []string{"knative-productivity-oncall@googlegroups.com"}
 )
 
 const (
-	yamlURL = "https://raw.githubusercontent.com/knative/test-infra/master/tools/monitoring/sample.yaml"
+	projectID = "knative-tests"
+
+	yamlURL      = "https://raw.githubusercontent.com/knative/test-infra/master/tools/monitoring/sample.yaml"
+	crierSubName = "test-infra-monitoring-sub"
```

Comment:
```suggestion
	subscriber = "test-infra-monitoring-sub"
```

Reply:
Updated to `subname` as this is not the actual subscriber object.